dangerzone/conversion/doc_to_pixels.py

```diff
@@ -326,16 +346,26 @@ def pdftoppm_progress_callback(line: bytes) -> None:
             timeout=timeout,
         )
 
-        self.update_progress("Converted document to pixels")
-
-        # Move converted files into /tmp/dangerzone
-        for filename in (
+        final_files = (
             glob.glob("/tmp/page-*.rgb")
             + glob.glob("/tmp/page-*.width")
             + glob.glob("/tmp/page-*.height")
-        ):
+        )
+
+        # XXX: Sanity check to avoid situations like #560.
+        if len(final_files) != 3 * num_pages:
+            raise errors.UnexpectedConversionError(
+                "The final number of pages does not correspond to the page count of the"
+                " original document"
```

Comment:
Now this part only makes sense in the containers version. Not sure why this didn't trigger the exception in Qubes now that it doesn't generate any files.

Should we add here a check to see if it's not running on Qubes?

Reply:
... damn. Yes, I guess I didn't test this part in Qubes. I'll add a fixup.

Reply:
Fixed in ed6722a73fbaac070aecd90c4b8fd2f3dccaf78a